python/satyaml/MEZNSAT.yml

```diff
@@ -11,7 +11,14 @@ transmitters:
     framing: AX.25 G3RUH
     data:
     - *tlm
-  9k6 BPSK downlink:
+  2k4 BPSK downlink:
+    frequency: 436.600e+6
+    modulation: BPSK
+    baudrate: 2400
+    framing: AX.25 G3RUH
+    data:
+    - *tlm
+9k6 BPSK downlink:
```

Comment:
Hi Jan, thanks for submitting this.

There is a problem with the format of the SatYAML file. The indentation (two spaces) in this line `9k6 BPSK downlink` is missing. As you can see, the [CI tests](http://eala.destevez.net:8010/#/builders/2/builds/469) complain about this:

```
11: Test command: /usr/bin/sh "/home/buildbot/worker/maketest/build/python/satyaml/qa_satyaml_test.sh"
11: Test timeout computed to be: 10000000
11: 
11: ======================================================================
11: ERROR: test_check_all_satyaml_files (__main__.qa_satyaml) (satyaml=PosixPath('/home/buildbot/worker/maketest/src/python/satyaml/MEZNSAT.yml'))
11: ----------------------------------------------------------------------
11: Traceback (most recent call last):
11:   File "/home/buildbot/worker/maketest/src/python/satyaml/qa_satyaml.py", line 19, in test_check_all_satyaml_files
11:     satyaml.yamlfiles.check_yaml(yml)
11:   File "/home/buildbot/worker/maketest/src/python/satyaml/satyaml.py", line 42, in check_yaml
11:     raise YAMLError(f'Unknown word {word} in {yml}')
11: satyaml.YAMLError: Unknown word 9k6 BPSK downlink in /home/buildbot/worker/maketest/src/python/satyaml/MEZNSAT.yml
11: 
11: ----------------------------------------------------------------------
11: Ran 1 test in 0.471s
11: 
11: FAILED (errors=1)
11/12 Test #11: qa_satyaml .......................***Failed    0.58 sec
```